ui/page/wallet_page.go

```diff
@@ -694,42 +694,47 @@ func (pg *WalletPage) walletAccountsLayout(gtx layout.Context, account *dcrlibwa
 	)
 }
 
-func (pg *WalletPage) backupSeedNotification(gtx layout.Context, listItem *walletListItem) layout.Dimensions {
+func (pg *WalletPage) backupSeedNotification(gtx layout.Context, index int) layout.Dimensions {
 	gtx.Constraints.Min.X = gtx.Constraints.Max.X
 	textColour := pg.Theme.Color.InvText
-	return layout.UniformInset(values.MarginPadding10).Layout(gtx, func(gtx C) D {
-		return layout.Flex{Axis: layout.Horizontal, Spacing: layout.SpaceBetween}.Layout(gtx,
-			layout.Rigid(func(gtx C) D {
-				return pg.walletAlertIcon.Layout(gtx)
-			}),
-			layout.Rigid(func(gtx C) D {
-				inset := layout.Inset{
-					Left: values.MarginPadding10,
-				}
-				return inset.Layout(gtx, func(gtx C) D {
-					return layout.Flex{Axis: layout.Vertical}.Layout(gtx,
-						layout.Rigid(func(gtx C) D {
-							txt := pg.Theme.Body2(values.String(values.StrBackupSeedPhrase))
-							txt.Color = textColour
-							return txt.Layout(gtx)
-						}),
-						layout.Rigid(func(gtx C) D {
-							txt := pg.Theme.Caption(values.String(values.StrVerifySeedInfo))
-							txt.Color = textColour
-							return txt.Layout(gtx)
-						}),
-					)
-				})
-			}),
-			layout.Flexed(1, func(gtx C) D {
-				inset := layout.Inset{
-					Top: values.MarginPadding5,
-				}
-				return inset.Layout(gtx, func(gtx C) D {
-					return layout.E.Layout(gtx, listItem.backupAcctBtn.Layout)
-				})
-			}),
-		)
+	return pg.clickableRow(gtx, pg.listItems[index].backupAcctClickable, func(gtx C) D {
```

Comment:
Remove redundant wrapper function
```suggestion
	return decredmaterial.Clickable(gtx, pg.listItems[index].backupAcctClickable, func(gtx C) D {
```